modules/gitops-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-3_{context}"]
+= Release notes for {gitops-title} 1.8.3
+
+{gitops-title} 1.8.3 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[IMPORTANT]
+====
+Before upgrading to {product-title} 4.13, move out of the `stable` channel.
+====
+
+[id="errata-updates-1-8-3_{context}"]
+== Errata updates
+
+=== RHEA-2023:112944 and RHBA-2023:112945 - {gitops-title} 1.8.3 security update advisory 
+
+Issued: 2023-04-17
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
+* link:https://access.redhat.com/errata/RHBA-2023:112945[RHBA-2023:112945]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

Comment:
```suggestion
* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled, and HPA parameters are correctly updated.  
link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

Reply:
@jaideepr97 Addressed in the revised PR. PTAL.